### DISCUSSION TITLE
Warn if you update a style that you currently have open in your editor

### DISCUSSION THREAD

**Discussion by RheingoldRiver** [stylus/stylus#2667] [Ideas]
If you have a style open and install an update, it doesn't immediately load the update, which makes sense in case you had unsaved changes or something. But, it should warn you of this situation so that you don't think the update is broken/not working/etc (I had this happen just now and complained to the style developer, and we were both confused for several minutes about what was going on).

**Comment by iChenLei** [comment]
@RheingoldRiver I think you should report your issue to https://github.com/openstyles/stylus ?

**Reply by RheingoldRiver** [reply]
Oh! Yes, sorry!!
